core/src/avm2/globals/flash/display/displayobject.rs

```diff
@@ -239,23 +239,38 @@ pub fn set_scale_x<'gc>(
     Ok(Value::Undefined)
 }
 
-/// Implements `filters`'s getter.
 pub fn filters<'gc>(
     activation: &mut Activation<'_, 'gc, '_>,
-    _this: Option<Object<'gc>>,
+    this: Option<Object<'gc>>,
     _args: &[Value<'gc>],
 ) -> Result<Value<'gc>, Error<'gc>> {
-    log::warn!("DisplayObject.filters getter - not yet implemented");
+    if let Some(dobj) = this.and_then(|this| this.as_display_object()) {
+        return Ok(ArrayObject::from_storage(activation, dobj.filters())?.into());
```

Comment:
Interesting, just checked that indeed it returns a new Array every time.
(`trace(this.filters === this.filters); // false`). Could you please add a test for this?

Reply:
Done and checked on flash debug